apps/services/utils.py

```diff
@@ -119,3 +135,25 @@ def check_file_existing(path: str) -> None:
             message=f"File with path {path} was removed or deleted",
             code=http_status.HTTP_409_CONFLICT,
         )
+
+
+def update_user_booking_hostel_data_by_user_request(
+    user_request_data: dataclass, user_booking_hostel_data: Row
```

Comment:
What does it mean type annotation of `:dataclass`? Better to provide real instance `<DATACLASS_NAME>` or `Type["<DATACLASS_NAME>"]`

Reply:
ok